src/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App: React.FC = () => {
+interface Movie {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+export const App: FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies: Movie[] = [...moviesFromServer]
+    .filter(({ title, description }: Movie) => {
+      const keyword = query.toLowerCase().trim();
+      const isInTitle = title.toLowerCase().includes(keyword);
+      const isInDescription = description.toLowerCase().includes(keyword);
```

Comment:
you share the same logic  - `toLowerCase` and `includes` between title and description comparison, create helper func for it